src/nvim/globals.h

```diff
@@ -1086,8 +1000,8 @@ EXTERN char psepcN INIT(= '/');            // abnormal path separator character
 EXTERN char pseps[2] INIT(= { '\\', 0 });  // normal path separator string
 #endif
 
-/* Set to TRUE when an operator is being executed with virtual editing, MAYBE
- * when no operator is being executed, FALSE otherwise. */
+// Set to bool when an operator is being executed with virtual editing, MAYBE
```

Comment:
Though a tri-state better not use `true`/`false`. Either use `TriState` or refactor to keep track of "no operator being executed" otherhow.